src/layers/ImageLayer.js

```diff
@@ -150,12 +150,15 @@ const ImageLayer = class extends CompositeLayer {
       modelMatrix,
       transparentColor
     } = this.props;
-    const { dtype, photometricInterpretation } = loader;
+    const { dtype } = loader;
     const { width, height, data, unprojectLensBounds } = this.state;
     if (!(width && height)) return null;
 
     const bounds = [0, height, width, 0];
-    if (isInterleaved(loader)) {
+    if (isInterleaved(loader.shape)) {
+      const {
+        meta: { photometricInterpretation }
+      } = loader;
```

Comment:
What happens if no `photometricInterpretation` is provided? Is there a default? This is not included in Zarr, so it would be good to have a default:

```javascript
const { photometricInterpretation = ?? } = loader.meta;
```

Reply:
Good call, I'll add a default.